numpy/core/_methods.py

```diff
@@ -165,6 +165,15 @@ def _mean(a, axis=None, dtype=None, out=None, keepdims=False):
 def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     arr = asanyarray(a)
 
+    # Fast path: avoid path that calls um.conjugate for complex input.
+    # The um.conjugate path is still hit for object arrays (thus cannot be
+    # removed).
+    if issubclass(arr.dtype.type, nt.complexfloating):
+        return um.add(
+            _var(arr.real, axis, dtype, out, ddof, keepdims),
+            _var(arr.imag, axis, dtype, out, ddof, keepdims)
+        )
```

Comment:
Reusing `out` here doesn't seem safe

Reply:
Good point - will look into it and think about adding new test(s) for this extra path.

In your opinion, is the change even worth it? I'm not sure how I feel about adding extra complexity for ~20% speedup (which is very system-dependent).

Reply:
Can you get the same speedup by adding the fast path right where `conjugate` is called?

Reply:
I had originally been attempting a solution along those lines - If there were a `conjmultiply` ufunc for instance that would do the trick but I don't know of any such function.

Reply:
How about
```
um.multiply(x.real, x.real, out=x.real)
um.multiply(x.imag, x.imag, out=x.imag)
x = np.add(x.real, x.imag)  # could use out=x.real, but may be slower
```

Reply:
I will try along those lines - thanks for the suggestion

Reply:
`um.add(um.multiply(x.real, x.real), um.multiply(x.imag, x.imag))`

This results in the speedup, but unfortunately still implicitly depends on the dtype of the input being complex - it fails on the test suite for object arrays with complex numbers in them

Reply:
I think special casing complex dtypes is ok, as long as you do it in the same place that the other types are specialized.

Reply:
Thanks for the feedback - the current approach (c049074) keeps all of the special cases based on dtypes in the same area of the code where all the other type-determination occurs.